FireRender.Maya.Src/FireRenderExportCmd.cpp

```diff
@@ -348,19 +350,20 @@ MStatus FireRenderExportCmd::doIt(const MArgList & args)
 		}
 
 		// process file path
-		MString fileName;
-		MString fileExtension = "rpr";
+		std::string fileName;
+		std::string fileExtension = "rpr";
 
 		// Remove extension from file name, because it would be added later
-		int fileExtensionIndex = filePath.rindexW("." + fileExtension);
+		size_t fileExtensionIndex = filePath.find("." + fileExtension);
 		bool fileExtensionNotProvided = fileExtensionIndex == -1;
+
 		if (fileExtensionNotProvided)
 		{
 			fileName = filePath;
 		}
 		else
 		{
-			fileName = filePath.substringW(0, fileExtensionIndex - 1);
```

Comment:
is it correct that -1 was removed

Reply:
different logic in MString and std::string